Secret-agency-service/src/test/java/cz/fi/muni/pa165/secretagency/service/AgentServiceTest.java

```diff
@@ -55,16 +61,16 @@ public void getAllAgentsTest() {
     /**
      * Test that agent is created
      */
-    @Test
-    public void createAgentTest() {
-        Agent agent = new Agent();
-        agent.setId(50L);
-        agent.setCodeName("Rum");
-        agent.setRank(AgentRankEnum.JUNIOR);
-        when(agentDao.save(agent)).thenReturn(agent);
-        Agent savedAgent = agentService.create(agent);
-        assertEquals(agent, savedAgent);
-    }
+//    @Test
```

Comment:
Toto asi odkomentovat nebo zahodit

Reply:
Já mám pocit, že jsem to zakomentoval kvůli tomu, že metoda create neexistuje.

Reply:
Moje chyba, nevšiml jsem si, že to, co jsi měl pojmenované jako create, je u mě save.